cpp/kiss_icp/3rdparty/eigen/eigen.cmake

```diff
@@ -33,6 +33,7 @@ set(EIGEN_BUILD_LAPACK OFF CACHE BOOL "Don't build lapack module")
 
 include(FetchContent)
 FetchContent_Declare(eigen URL https://github.com/nachovizzo/eigen/archive/refs/tags/3.4.90.tar.gz)
+FetchContent_GetProperties(eigen)
 if(NOT eigen_POPULATED)
```

Comment:
This basically means this was literally `if (TRUE)` since the beginning of time

Reply:
Yeees, when I found out my brain just got bananas

Reply:
But it is now done